README.md

```diff
@@ -26,21 +26,10 @@ Pagination is handled by either Kaminari or WillPaginate. Make sure you have one
 2. `cd library-cms`
 3. `bundle install && yarn install`
 4. `docker-compose up -d`
-5. `rake db:create && rake db:migrate`
-6. change default credentials for your development environment by editing `config/initializers/comfortable_mexican_sofa.rb`:
-```
-    ComfortableMexicanSofa::AccessControl::AdminAuthentication.username = 'dev-username'
-    ComfortableMexicanSofa::AccessControl::AdminAuthentication.password = 'dev-password'
-```
-    also add default credentials in 'app/controllers/profiles_controller.rb' :
-    http_basic_authenticate_with name: Rails.application.secrets.cms_user, password: Rails.application.secrets.cms_password, except: [:index, :show]
-    replace Rails.application.secrets.cms_user with "dev-username" and Rails.application.secrets.cms_password with "dev-password"
-    
-7. `rails s`
-8. visit http://localhost:3000/admin - You'll be prompted to enter username and password (use the ones you created in step 6)
-9. In the admin area create a site called "ualberta-libraries"
-10. Seed database: `rake 'comfy:cms_seeds:import[library-cms-seeds, ualberta-libraries]'` may take a couple of minutes
-11. visit http://localhost:3000 and you will see the library homepage. 
+5. `bin/rails db:setup`
```

Comment:
I had trouble with using exactly this command.  `bundle exec rails db:setup` worked

```
tricia@LTLNXLAPTC1:~/Code/library-cms$ ./bin/rails db:setup
Lost connection to server at 'handshake: reading initial communication packet', system error: 11
Couldn't create 'cms_development' database. Please check your configuration.
bin/rails aborted!
ActiveRecord::ConnectionNotEstablished: Lost connection to server at 'handshake: reading initial communication packet', system error: 11


Caused by:
Mysql2::Error::ConnectionError: Lost connection to server at 'handshake: reading initial communication packet', system error: 11

Tasks: TOP => db:setup => db:create
(See full trace by running task with --trace)
tricia@LTLNXLAPTC1:~/Code/library-cms$ bundle exec rails db:setup
Created database 'cms_development'
Created database 'cms_test'
Importing CMS Seed data from Folder [library-cms-seeds] to Site [ualberta-libraries] ...
```

Reply:
`bin/rails db:setup` is still an obstacle for me.  But maybe `bundle exec` is common knowledge and doesn't need additional explanation.